fix-ebpf-for-alpine3.13.patch

```diff
@@ -0,0 +1,94 @@
+@@ -97,7 +97,7 @@ index c53c542..8244f77 100644
+ 
+  # Link against LLVM libraries
+ -target_link_libraries(bcc-shared ${bcc_common_libs_for_s})
+-+target_link_libraries(bcc-shared ${bcc_common_libs_for_s} -Wl,--wrap=exp -Wl,--wrap=log -Wl,--wrap=pow -Wl,--wrap=exp2 -Wl,--wrap=log2)
+++target_link_libraries(bcc-shared ${bcc_common_libs_for_s})
+  target_link_libraries(bcc-static ${bcc_common_libs_for_a} bcc-loader-static)
+  set(bcc-lua-static ${bcc-lua-static} ${bcc_common_libs_for_lua})
+ 
+diff --git a/pkg/ebpf/compiler/compiler.go b/pkg/ebpf/compiler/compiler.go
+index 804fee14e..1acad984c 100644
+--- a/pkg/ebpf/compiler/compiler.go
++++ b/pkg/ebpf/compiler/compiler.go
+@@ -7,7 +7,6 @@ package compiler
+ #cgo LDFLAGS: -L/opt/datadog-agent/embedded/lib
+ #cgo LDFLAGS: -lLLVMXRay -lLLVMWindowsManifest -lLLVMTableGen -lLLVMSymbolize -lLLVMDebugInfoPDB -lLLVMOrcJIT -lLLVMOrcError -lLLVMJITLink -lLLVMObjectYAML -lLLVMMIRParser -lLLVMMCJIT -lLLVMMCA -lLLVMLTO -lLLVMPasses -lLLVMCoroutines -lLLVMObjCARCOpts -lLLVMipo -lLLVMInstrumentation -lLLVMVectorize -lLLVMLinker -lLLVMIRReader -lLLVMAsmParser -lLLVMFrontendOpenMP -lLLVMExtensions -lLLVMLineEditor -lLLVMLibDriver -lLLVMGlobalISel -lLLVMFuzzMutate -lLLVMInterpreter -lLLVMExecutionEngine -lLLVMRuntimeDyld -lLLVMDWARFLinker -lLLVMDlltoolDriver -lLLVMOption -lLLVMDebugInfoGSYM -lLLVMCoverage -lLLVMCFGuard -lLLVMBPFDisassembler -lLLVMMCDisassembler -lLLVMBPFCodeGen -lLLVMSelectionDAG -lLLVMAsmPrinter -lLLVMDebugInfoDWARF -lLLVMCodeGen -lLLVMTarget -lLLVMScalarOpts -lLLVMInstCombine -lLLVMAggressiveInstCombine -lLLVMTransformUtils -lLLVMBitWriter -lLLVMAnalysis -lLLVMProfileData -lLLVMObject -lLLVMTextAPI -lLLVMBitReader -lLLVMCore -lLLVMRemarks -lLLVMBitstreamReader -lLLVMBPFAsmParser -lLLVMMCParser -lLLVMBPFDesc -lLLVMMC -lLLVMDebugInfoCodeView -lLLVMDebugInfoMSF -lLLVMBinaryFormat -lLLVMBPFInfo -lLLVMSupport -lLLVMDemangle
+ #cgo LDFLAGS: -lz -ldl -lm -lrt -static-libstdc++
+-#cgo LDFLAGS: -Wl,--wrap=exp -Wl,--wrap=log -Wl,--wrap=pow -Wl,--wrap=log2 -Wl,--wrap=log2f
+ #cgo CXXFLAGS: -I/opt/datadog-agent/embedded/include -std=c++14 -fno-exceptions -fno-rtti -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -DLLVM_MAJOR_VERSION=11
+ #cgo CPPFLAGS: -I/opt/datadog-agent/embedded/include -D_GNU_SOURCE -D_DEBUG -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -DLLVM_MAJOR_VERSION=11
+ 
+diff --git a/pkg/ebpf/compiler/wrapper.h b/pkg/ebpf/compiler/wrapper.h
+index 31b00820c..51f1ae231 100644
+--- a/pkg/ebpf/compiler/wrapper.h
++++ b/pkg/ebpf/compiler/wrapper.h
+@@ -48,8 +48,8 @@ float __wrap_ ## func (float x) {                               \
+   return __ ## func ## _prior_glibc(x);                         \
+ }
+ 
+-symver_wrap_d1(exp)
+-symver_wrap_d1(log)
+-symver_wrap_d2(pow)
+-symver_wrap_d1(log2)
+-symver_wrap_f1(log2f)
++//symver_wrap_d1(exp)
++//symver_wrap_d1(log)
++//symver_wrap_d2(pow)
++//symver_wrap_d1(log2)
++//symver_wrap_f1(log2f)
```

Comment:
These are for removing version dependency of glibc. (It's not a problem on musl-libc based system.)